database/migrations/2022_02_23_064927_add_column_in_reactions.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddColumnInReactions extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('reactions', function (Blueprint $table) {
+            $table->integer('comment_id');
```

Comment:
Tìm hiểu và làm theo kiểu polymorphic, ko thêm cột kiểu này.
https://laravel.com/docs/9.x/eloquent-relationships#one-to-many-polymorphic-relations